Cargo.toml

```diff
@@ -3,47 +3,48 @@ panic = 'unwind'
 
 [workspace]
 members = [
-    'node',
-    'runtime/hydradx',
-    'pallets/omnipool',
-    'pallets/dca',
-    'primitives',
-    'utils/build-script-utils',
-    'integration-tests',
-    'pallets/circuit-breaker',
-    'pallets/omnipool-liquidity-mining',
-    'scraper',
-    'traits',
-    'pallets/relaychain-info',
-    'pallets/route-executor',
-    'pallets/transaction-multi-payment',
-    'pallets/asset-registry',
-    'runtime/adapters',
-    'pallets/collator-rewards',
-    'pallets/transaction-pause',
-    'pallets/ema-oracle',
-    'pallets/liquidity-mining',
-    'pallets/currencies',
+  'node',
+  'runtime/hydradx',
+  'pallets/omnipool',
```

Comment:
Was this formatted wrongly b4?

Reply:
i think yes, it was 8 instead of 4 spaces

Reply:
i think yes, it should be 4 spaces instead of 8. Or am i using some wrong fmt config?